opentech/static_src/src/app/src/components/ReviewsBlock/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Review = ({ review }) => {
+    const { reviewUrl, author, score, recommendation } = review
+
+    return (
+        <p>
+            <a href={reviewUrl}>
+                {author} - {score} - {recommendation.display}
+            </a>
+        </p>
+    )
+}
+
+Review.propTypes = {
+    review: PropTypes.shape({
```

Comment:
Can we not just unpack the review data into the prop directly without the intermediary review level?